src/index.js

```diff
@@ -0,0 +1,152 @@
+
+let t = {
+    temperature: 60
+};
+
+const kelvinToFahrenheit = (t) => {
+    tempretureInFahrenhait = 1.8 * (t - 273) + 32;
+    return tempretureInFahrenhait.toFixed(2);
+}
+
+const temperatureColor = () => {
+    const temperature = document.querySelector("#temperatureCount");
+    switch(true) {
+        case (t.temperature >= 80):
+            temperature.style.color = 'red';
```

Comment:
If you made this a class (say `hot`) and then styled that then you'd better follow [separation of concerns](https://en.wikipedia.org/wiki/Separation_of_concerns).